### DISCUSSION TITLE
CTD with new update 6 from MS

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6048] [General]
Hi,

since yesterday the new update for MS 2020 is available. First flight today. so far so gut. Everything was looking fine. Till my descent clearence:-( .... I was at FL380 and requested FL300. My AI copilot was answering ATC, I wanted to tune in the new FL. At this moment, sound was gone and it crashed to desktop.... Well, I think it is hard to say where the issue came from. Perhaps some other users will have the same issue, please report it here ..

Greets Klaus

**Comment by CptBayer** [comment]
I am using dev mode .... 

![grafik](https://user-images.githubusercontent.com/76531893/138108682-d59a96e2-f84d-4d6e-aaa9-e4f32c884c0d.png)

**Reply by tracernz** [reply]
> ![grafik](https://user-images.githubusercontent.com/76531893/138108682-d59a96e2-f84d-4d6e-aaa9-e4f32c884c0d.png)

That information is not correct.

Unfortunately the causes of CTDs are numerous (scenery, add-ons, aircraft, etc.), and only Microsoft has the data to figure out what causes them. The reason you will see many crash reports for A32NX is because a large proportion of MSFS flights are done with A32NX.

**Reply by CptBayer** [reply]
Hi to New Zealand :-) ..

that´s to bad that only Microsoft has the data to figure out the reason... So it is like a search for the needle in the hay.... ahhrggg .... I hate it ... But I had only the FBW bus in the community folder. As I wrote above, hard to find the reason. I remember there were issues with ATC and sound that causes CTD´s in the past .... hopefully microsoft will implement error messages as we know it from other versions ...

greets Klaus

**Reply by Paytheon** [reply]
What I've noticed is there's a rudder issue on landing since update 6. The plane is EXTREMELY unstable when landing. Even Auto Landing struggles with landing. I experience this instability when landing at KSEA and KPDX. So far I've not experienced a CTD using Dev.

**Reply by mreza1st** [reply]
A lot of times the servers are the reason for CTDs. On a busy day, right after update, CTDs are more frequent. I normally don't get any CTDs but I do when the servers are busy after update, or before an update.
